src/dbnode/storage/index/types.go

```diff
@@ -186,6 +209,9 @@ type AggregateResultsOptions struct {
 
 	// Optional param to filter aggregate values.
 	TermFilter AggregateTermFilter
+
+	// Type determines what result is required.
+	Type AggregationType
```

Comment:
can you delete AggregateResultsOptions now?

Reply:
I wanted to keep it as similar as possible to the regular `Query` path which does a similar thing when it breaks out the options relating to the result from those relating to the query parameters here